parts/kubernetesmastercustomdata.yml

```diff
@@ -7,8 +7,30 @@ packages:
  - jq
  - traceroute
 
+disk_setup:
+  /dev/sdc:
+      table_type: gpt
+      layout: true
+      overwrite: true
+
+fs_setup:
+- special:
+  cmd: /sbin/mkfs.ext4 /dev/sdc1 -F -E lazy_itable_init=1,lazy_journal_init=1
```

Comment:
I don't think this is fragile based on my understanding and reading of this. Any reason why it would be?

This is lazy formatting where enough blocks are written but complete disk is not formatted with the rest happening in the background when its first mounted. It was taking about 8 mins for the formatting to finish for a 128 GB disk that caused Linux Agent to time out.

Another way to fix this would be to not do time consuming operations like setting up etcd and apt-get during cloud init. Instead kick of another script during cloud init that runs in the background and read the status during CSE run. 

However, at this time I don't see this as a blocking issue or we could go back to using a smaller etcd disk.